app/lists.rb

```diff
@@ -0,0 +1,43 @@
+module Listing
+  extend Discordrb::Commands::CommandContainer
+  
+  command :WingInitilise, description: "Initilise the array. In case of derps." do |event|
+	wing_list = Array.new
+  end
+
+  command :WingList, description: "Lists those who are requesting wing." do |event| #Simple, return the text file's contents. May consider making this a array. Also, admin lock this.
```

Comment:
Comment above the command please, to avoid insane line wrapping? I'm pretty bad about it myself, but I try not to be. :/